src/Tasks/Hash.cs

```diff
@@ -37,61 +35,102 @@ public class Hash : TaskExtension
         [Output]
         public string HashResult { get; set; }
 
+        private static readonly char ItemSeparatorCharacter = '\u2028';
+
+        private static readonly Encoding encoding = Encoding.UTF8;
+
+        private static readonly byte[] ItemSeparatorCharacterBytes = encoding.GetBytes(new char[] { ItemSeparatorCharacter });
```

Comment:
Please rename it to `s_ItemSeparatorCharacterBytes` to follow naming rules for private statics.